specs/MultiProfile.md

```diff
@@ -0,0 +1,146 @@
+API spec for multiple profile support
+===
+
+# Background
+
+Currently, all WebView2s can only use one fixed Edge profile in the browser process, which is
+normally the **Default** profile or the profile specified by the **--profile-directory** command
+line switch. Regarding this we have got a bunch of requests to support multiple profiles, so we're
+adding these new APIs.
+
+With this you can have different WebView2s running with separate profiles under a given user data
+directory, which means separate cookies, user preference settings, and various data storage etc.,
+to enable you to build a more wonderful experience for your application.
+
+# Examples
+<!-- TEMPLATE
```

Comment:
Missing sample code. This could be the same as the sample code you add to the sample app.

Reply:
Done.